cmd/stringer/golden_test.go

```diff
@@ -397,6 +401,80 @@ func (i Token) String() string {
 }
 `
 
+const toLower_in = `type Day int
```

Comment:
warning: don't use underscores in Go names; const toLower_in should be toLowerIn (golint)